test/public/defaults.js

```diff
@@ -697,3 +711,19 @@ module.exports.expectLink = async (element, selector, { href, innerText }) => {
  * @return {boolean} true if format is correct, false otherwise
  */
 module.exports.validateDate = (date, format = 'DD/MM/YYYY hh:mm:ss') => !isNaN(dateAndTime.parse(date, format));
+
+/**
+ * Return the selector for all the inputs composing a period inputs selectors
+ *
```

Comment:
Return selectors for all inputs composing a period input ?

Reply:
Fixed